index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <nav class ="main-nav">
+        <div class="logo">
+        <a href="#"><p>Worash Aboche</p></a>  
+        </div>
+         <a href="#"> <img src ="./images/Icon-Menu.svg" alt="menu" class="menu"> </a>
+        </nav>
```

Comment:
- [ ] Please container of this section should be a `header` tag
- [ ] Please the hamburger button should be on the right and have a `nav` tag, there is no need to this a tag that contains the hamburger button.